modules/core/04-channel/keeper/upgrade.go

```diff
@@ -329,7 +327,7 @@ func (k Keeper) WriteUpgradeAckChannel(ctx sdk.Context, portID, channelID string
 
 	k.SetUpgrade(ctx, portID, channelID, upgrade)
 
-	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "previous-state", previousState, "new-state", types.ACKUPGRADE.String())
+	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "state", channel.State.String())
```

Comment:
possible nit: I did the same thing for Confirm but placed it inside the `if` where the state change would occur.

Reply:
here we don't actually know what the previous state actually was, because `startFlush` modifies it within the Ack handler. This means it was either OPEN or FLUSHING but we don't know which 🤔 I figured it would be enough to just log the current state after modification.